scripts/smoke-test/teaser_smoke_test.py

```diff
@@ -0,0 +1,177 @@
+import time
+import argparse
+from utils.android_view_helper import AndroidViewHelper
+from utils.command_runner import CommandRunner
+from utils.device_executor import DeviceExecutor
+
+PROJECT_FOLDER = '../../'
+BUILD_RELEASE_COMMAND = './gradlew clean :app:assembleRelease'
+TEASER_APK_FILE_PATH = PROJECT_FOLDER + 'app/build/outputs/apk/release/app-arm64-v8a-release.apk'
+OUTPUT_FOLDER = PROJECT_FOLDER + 'app/build/outputs/smoke-test'
+PACKAGE_NAME = 'com.mapbox.vision.teaser'
+START_ACTIVITY_NAME = 'com.mapbox.vision.examples.activity.main.MainActivity'
+
+AR_ROUTING_SCREEN = 'ar_routing'
+
+SCREEN_ID_MAP = {
+    'sign_detection': 'sign_detection_container',
+    'segmentation': 'segm_container',
+    'object_detection': 'det_container',
+    'lane_detection': 'line_detection_container',
+    'safety_mode': 'distance_container',
+    AR_ROUTING_SCREEN: 'ar_navigation_button_container',
+}
+
+LOG_ADB_COMMANDS = True
+LOG_ACTIONS = True
+USE_LOG_CHECK = True  # Set False for LG G6
+DEFAULT_TIME_BEFORE_SCREENSHOT_MS = 4000
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-b", dest="need_build_apk", help="branch for analysis", default=False,
+                        type=lambda x: (str(x).lower() == 'true'))
+    parser.add_argument("-t", dest="time_before_screenshot", help="file name with config",
+                        default=DEFAULT_TIME_BEFORE_SCREENSHOT_MS)
+    return parser.parse_args()
+
+
+def log_message_to_console(message):
+    if LOG_ACTIONS:
+        print "*" * 80
+        print '*   {:75}*'.format(message)
+        print "*" * 80
+        print "\n"
+
+
+def build_release_apk(args):
+    if args.need_build_apk:
+        command_runner = CommandRunner(LOG_ADB_COMMANDS)
+        log_message_to_console("Build Teaser release apk file")
+        command_runner.execute_command_with_answer(BUILD_RELEASE_COMMAND, PROJECT_FOLDER)
+
+
+def get_device_list():
+    command_runner = CommandRunner(LOG_ADB_COMMANDS)
+    str_ = command_runner.execute_command_with_answer('adb devices')
+    devices_info = str_[str_.index('List of devices attached') + len('List of devices attached') + 1:]
```

Comment:
Try string.partition. It will be something like 
```suggestion
    _, _, devices_info = str_.partition('List of devices attached')
```
Maybe with extra space symbol

Reply:
done